src/torchmetrics/classification/accuracy.py

```diff
@@ -511,3 +511,15 @@ def __new__(  # type: ignore[misc]
                 )
             return MultilabelAccuracy(num_labels, threshold, average, **kwargs)
         raise ValueError(f"Not handled value: {task}")
+
+    def update(self, *args: Any, **kwargs: Any) -> None:
+        """Update metric state."""
+        raise NotImplementedError(
+            f"{self.__class__.__name__} metric does not have a global `update` method. Use the task specific metric."
+        )
+
+    def compute(self) -> None:
+        """Compute metric."""
+        raise NotImplementedError(
+            f"{self.__class__.__name__} metric does not have a global `compute` method. Use the task specific metric."
+        )
```

Comment:
maybe create one more inter-level so to say `class _ClassifMetric(Metric)` which would hold the duplicated methods `update` and `compute`?

Reply:
Done :)